ravedude/src/board.rs

```diff
@@ -199,6 +200,35 @@ impl Board for ArduinoLeonardo {
     }
 }
 
+
+struct ArduinoMega1280;
+
+impl Board for ArduinoMega1280 {
+    fn display_name(&self) -> &str {
+        "Arduino Mega 1280"
+    }
+
+    fn needs_reset(&self) -> Option<&str> {
+        None
+    }
+
+    fn avrdude_options(&self) -> avrdude::AvrdudeOptions {
+        avrdude::AvrdudeOptions {
+            programmer: "arduino",
+            partno: "atmega1280",
+            baudrate: Some(57600),
+            do_chip_erase: false,
+        }
+    }
+
+    fn guess_port(&self) -> Option<anyhow::Result<std::path::PathBuf>> {
+        Some(find_port_from_vid_pid_list(&[
+            (0x0403,0x6001), // FTDI FT232R USB UART
+        ]))
```

Comment:
Please change this as follows:

```suggestion
        Some(Err(anyhow::anyhow!("Not able to guess port")))
```

As this board only shows up as a generic FTDI chip, there is too much ambiguity here for performing automatic detection :/  You'll have to pass the port using the `RAVEDUDE_PORT` environment variable for now... In the future I'd like to add a more user-friendly mechanism for this, but until then, I don't think we should auto-detect the generic chips as certain boards.

Reply:
Done, 

I also added a README to the mega1280 examples with instructions telling hoe to pass the port to ravedude